packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,308 @@
+       * The common additional properties for image in mobile devices
+       */
+      commonImageprops: ImageProperties;
+    }
+
+    /**
+     *  All properties in ImageProperties are optional and have default values in the platform
+     *  Additional properties for image in mobile devices
+     * @beta
+     */
+    export interface ImageProperties {
+      /**
+       * Optional; indicate if inking on the selected Image is allowed or not
+       * Default value is false
+       */
+      shouldAllowInkingOnImages?: boolean;
+
+      /**
+       * Optional; indicate if putting text stickers on the selected Image is allowed or not
+       * Default value is false
+       */
+      shouldAllowTextStickersOnImages?: boolean;
+
+      /**
+       * Optional; indicate if the filter mode is enabled on the selected image
+       * Default value is false
+       */
+      enableFilterOnImages?: boolean;
+
+      /**
+       * Optional; Lets the developer specify the image output formats, more than one can be specified.
+       * Default value is Image.
+       */
+      imageFormatsAllowedToBeSelected?: ImageOutputFormats[];
+    }
+
+    /**
+     * Specifies the image output formats.
+     * @beta
+     */
+    export enum ImageOutputFormats {
+      /** Outputs image.  */
+      Image = 1,
+      /** Outputs pdf. */
+      PDF = 2,
+    }
+    const errorInvalidSource = 'Invalid input source.';
+    const errorInvalidCount = 'Invalid input count: Must supply the valid image counts (limit to 10).';
+    const errorInvalidResponse = 'Invalid response: Received more images than the specified max limit in the response.';
+    /**
+     * Capture one or multiple image(s) using camera.
+     * @beta
+     * @param imageInputs - The input params to customize the image(s) to be captured
+     * @returns Promise that will resolve with {@link VisualMediaFile[]} object or reject with an error.
+     * @throws INVALID_ARGUMENTS Error if imageInputs is null or imageInputs.maxVisualMediaCount is greater than 10 or lesser than 1.
+     */
+    export async function captureImages(cameraImageInputs: CameraImageProperties): Promise<VisualMediaFile[]> {
+      ensureInitialized(runtime, FrameContexts.content, FrameContexts.task);
+      if (!isSupported()) {
+        throw errorNotSupportedOnPlatform;
+      }
+      if (
+        !cameraImageInputs ||
+        cameraImageInputs.maxVisualMediaCount > 10 ||
+        cameraImageInputs.maxVisualMediaCount < 1
+      ) {
+        throw errorInvalidCount;
+      }
+      if (cameraImageInputs.sourceProps.source != Source.Camera) {
+        throw errorInvalidSource;
+      }
+      // waiting the response from hub-SDK
+      const files = await sendAndHandleSdkError<VisualMediaFile[]>(
+        'visualMedia.image.captureImages',
+        cameraImageInputs,
+      );
+      // to ensure the number of images in the response is within the maximum limit.
+      if (files.length > cameraImageInputs.maxVisualMediaCount) {
```

Comment:
If we get more than 10 images don't we give an option to developer to take first 10 images or do batch processing of the images? Also, if we are getting valid response, throwing error seems not logical to me.

Reply:
That's a great point. That's from the PM decision: if we received more than 10 images, we will directly throw an error. Since any changes from PM side will effect the current Phase 0 deadline. I will make sure I mentioned your point to our PM for future improvement.

Reply:
Isn't this unnecessary use of the API usage if we are getting the response we should not waste it and avoid multiple calls.

Reply:
> Isn't this unnecessary use of the API usage if we are getting the response we should not waste it and avoid multiple calls.

I can't deny that, and I really appreciate your perspective on batch processing of the images. 
However, I'd like to retain the current design of the visualMedia API for Phase 0. This is because introducing batch processing of the images would entail a completely new design, and we'd need PM input to discuss the upper limits. For example, what if a user passes in 100 images, but the maxVisualMediaCount is set to 2 by the app developers? How would we handle image errors during batching? Would we want to dismiss the image selection UI in the main flow of the host app during image batching?

Reply:
Just to close the loop on this, the three of us discussed offline why it would be better to fail fast in this scenario and all agree that that is the right solution here.